commands/instances.go

```diff
@@ -200,9 +201,31 @@ func UpdateIndex(ctx context.Context, req *rpc.UpdateIndexReq, downloadCB Downlo
 	}
 
 	indexpath := paths.New(configuration.Settings.GetString("directories.Data"))
+	jsonPaths := []string{}
+	jsonPaths = append(jsonPaths, configuration.Settings.GetStringSlice("board_manager.additional_paths")...)
+
+	for _, x := range jsonPaths {
```

Comment:
You're not using `jsonPaths` anywhere else other than the `for`, you could do this to simplify:
```
for _, x := range configuration.Settings.GetStringSlice("board_manager.additional_paths") {
```

Same thing for the other in `createInstance()`.